### PR TITLE
docs: minor corrections to Networking Libraries reference

### PR DESCRIPTION
**Pull request by usaoc**
##### Checklist
- [x] documentation

### CODE REVIEW

File: pkgs/net-doc/net/scribblings/http-client.scrbl [hunk 1]
```diff
@@ -204,13 +204,18 @@ response, which is why there is no @racket[#:closed?] argument like
...
  @itemize[
-   @item{If @racket[ssl?] was @racket[#f] then @racket[#f]. Otherwise
-     an @racket[ssl-client-context?] that has been negotiated with the
-     target.
-
-     If @racket[ssl?] was a protocol symbol, then a new
-     @racket[ssl-client-context?] is created, otherwise the current
-     value of @racket[ssl?] is used.}
+   @item{
+     If @racket[ssl?] is @racket[#f], returns @racket[#f], otherwise
```
Comment (sorawee) [inline pkgs/net-doc/net/scribblings/http-client.scrbl:208]: It's kinda weird to say "returns" here. I like the old text better. The first bullet point is about the first value of the four values. There's no function that returns this single value specifically.

Comment (usaoc) [reply pkgs/net-doc/net/scribblings/http-client.scrbl:208]: The original phrasing read like an `if` expression, not an *English* sentence.  It took me some puzzling to understand what it was talking about.  A better phrasing is welcome, but it should read like an Enflish sentence.

Comment (sorawee) [reply pkgs/net-doc/net/scribblings/http-client.scrbl:208]: Perhaps replace "returns ..." with "the (first) value is ..."

Comment (usaoc) [reply pkgs/net-doc/net/scribblings/http-client.scrbl]: Rephrased this part (actually, the whole list).

**Pull request merged**